### DISCUSSION TITLE
Nice protection

### DISCUSSION THREAD

**Discussion by Metabaron1** [simple-login/app#1736] [Kind words ❤️]
Hi

i just got this uncomon personalized attack:

	MAIL FROM:<4jhmayftedplbbw@svremya.ru>
...
	RCPT TO:<postmaster@mydomain>
	RCPT TO:<abuse@mydomain>
	RCPT TO:<info@mydomain>
	RCPT TO:<admin@mydomain>
	RCPT TO:<webmaster@mydomain>
	RCPT TO:<mailer-daemon@mydomain>
	RCPT TO:<support@mydomain>
	RCPT TO:<sales@mydomain>
	RCPT TO:<contact@mydomain>
	RCPT TO:<all@mydomain>
	RCPT TO:<mail@mydomain>
	RCPT TO:<hostmaster@mydomain>
	RCPT TO:<marketing@mydomain>
	RCPT TO:<noreply@mydomain>
	RCPT TO:<hello@mydomain>
	RCPT TO:<office@mydomain>
	RCPT TO:<news@mydomain>
	RCPT TO:<root@mydomain>
	RCPT TO:<accounts@mydomain>
	RCPT TO:<nobody@mydomain>
	RCPT TO:<administrator@mydomain>
	RCPT TO:<service@mydomain>
	RCPT TO:<test@mydomain>
	RCPT TO:<daemon@mydomain>
	RCPT TO:<hr@mydomain>
	RCPT TO:<uucp@mydomain>
	RCPT TO:<security@mydomain>
	RCPT TO:<lp@mydomain>
	RCPT TO:<accounting@mydomain>
	RCPT TO:<david@mydomain>
	RCPT TO:<adm@mydomain>
	RCPT TO:<noc@mydomain>
	RCPT TO:<chris@mydomain>
	RCPT TO:<no-reply@mydomain>
	RCPT TO:<contato@mydomain>
	RCPT TO:<manager@mydomain>
	RCPT TO:<alex@mydomain>
	RCPT TO:<john@mydomain>
	RCPT TO:<www@mydomain>
	RCPT TO:<jobs@mydomain>
	RCPT TO:<newsletter@mydomain>
	RCPT TO:<usenet@mydomain>
	RCPT TO:<finance@mydomain>
	RCPT TO:<games@mydomain>
	RCPT TO:<kontakt@mydomain>
	RCPT TO:<operator@mydomain>
	RCPT TO:<team@mydomain>
	RCPT TO:<orders@mydomain>
	RCPT TO:<mike@mydomain>
	RCPT TO:<billing@mydomain>
	RCPT TO:<michael@mydomain>
	RCPT TO:<bin@mydomain>
	RCPT TO:<amanda@mydomain>
	RCPT TO:<help@mydomain>
	RCPT TO:<mailman-bounces@mydomain>
	RCPT TO:<fax@mydomain>
	RCPT TO:<system@mydomain>
	RCPT TO:<mark@mydomain>
	RCPT TO:<it@mydomain>
	RCPT TO:<daniel@mydomain>
	RCPT TO:<man@mydomain>
	RCPT TO:<ftp@mydomain>
	RCPT TO:<paul@mydomain>
	RCPT TO:<social@mydomain>
	RCPT TO:<apache@mydomain>
	RCPT TO:<web@mydomain>
	RCPT TO:<www-data@mydomain>
	RCPT TO:<steve@mydomain>
	RCPT TO:<comercial@mydomain>
	RCPT TO:<peter@mydomain>
	RCPT TO:<list@mydomain>
	RCPT TO:<careers@mydomain>
	RCPT TO:<paypal@mydomain>
	RCPT TO:<matt@mydomain>
	RCPT TO:<contacto@mydomain>
	RCPT TO:<biuro@mydomain>
	RCPT TO:<shop@mydomain>
	RCPT TO:<staff@mydomain>
	RCPT TO:<james@mydomain>
	RCPT TO:<tech@mydomain>
	RCPT TO:<dev@mydomain>
	RCPT TO:<anna@mydomain>
	RCPT TO:<events@mydomain>
	RCPT TO:<tom@mydomain>
	RCPT TO:<media@mydomain>
	RCPT TO:<financeiro@mydomain>
	RCPT TO:<post@mydomain>
	RCPT TO:<ventas@mydomain>
	RCPT TO:<enquiries@mydomain>
	RCPT TO:<andrew@mydomain>
	RCPT TO:<sarah@mydomain>
	RCPT TO:<wordpress@mydomain>
	RCPT TO:<administracion@mydomain>
	RCPT TO:<reply@mydomain>
	RCPT TO:<kevin@mydomain>
	RCPT TO:<jason@mydomain>
	RCPT TO:<order@mydomain>
	RCPT TO:<design@mydomain>
	RCPT TO:<ben@mydomain>
	RCPT TO:<oyre4289tuvu@mydomain>
...
	554 5.7.1 Service unavailable; Client host [58.62.85.113] blocked using bl.spamcop.net; Blocked - see https://www.spamcop.net/bl.shtml?58.62.85.113
	554 5.7.1 Service unavailable; Client host [58.62.85.113] blocked using bl.spamcop.net; Blocked - see https://www.spamcop.net/bl.shtml?58.62.85.113
	554 5.7.1 Service unavailable; Client host [58.62.85.113] blocked using bl.spamcop.net; Blocked - see https://www.spamcop.net/bl.shtml?58.62.85.113
	554 5.7.1 Service unavailable; Client host [58.62.85.113] blocked using bl.spamcop.net; Blocked - see https://www.spamcop.net/bl.shtml?58.62.85.113
...
	421 4.7.0 mydomain Error: too many errors

I don't have any of these default/standard recipients but it's cool you included these Spamcop/Spamhaus checks ;)
Thanks for your job

**Comment by nguyenkims** [comment]
Glad that spamcop/spamhaus helps!
